dataprofiler/profilers/float_column_profile.py

```diff
@@ -105,23 +105,39 @@ def _get_float_precision(cls, df_series):
         :return: string representing its precision print format
```

Comment:
Add df_series description for required input

Reply:
updated